Sources/SwiftGen/Config/Config.swift

```diff
@@ -200,17 +200,23 @@ extension Config {
 extension Config {
   enum Error: Swift.Error, CustomStringConvertible {
     case missingEntry(key: String)
-    case wrongType(key: String?, expected: String, got: Any.Type)
+    case multipleErrors([Swift.Error])
     case pathNotFound(path: Path)
+    case unknownAction(name: String)
+    case wrongType(key: String?, expected: String, got: Any.Type)
 
     var description: String {
       switch self {
       case .missingEntry(let key):
         return "Missing entry for key \(key)."
-      case .wrongType(let key, let expected, let got):
-        return "Wrong type for key \(key ?? "root"): expected \(expected), got \(got)."
+      case .multipleErrors(let errors):
+        return errors.map { $0.localizedDescription }.joined(separator: "\n")
       case .pathNotFound(let path):
         return "File \(path) not found."
+      case .unknownAction(let name):
+        return "Action `\(name)` does not exist."
```

Comment:
s/Action/Parser/?

Reply:
Actually did that on purpose, as the `config lint` also outputs "action" instead of parser.

Reply:
Right. Shouldn't we try to uniformise tho to avoid losing the users by using different terms? Not sure what's best 🤷🏻